third_party/terraform/tests/resource_sourcerepo_repository_test.go

```diff
@@ -6,24 +6,19 @@ import (
 
 	"github.com/hashicorp/terraform/helper/acctest"
 	"github.com/hashicorp/terraform/helper/resource"
-	"github.com/hashicorp/terraform/terraform"
 )
 
-func TestAccSourceRepoRepository_basic(t *testing.T) {
+func TestAccSourcerepoRepository_basic(t *testing.T) {
```

Comment:
Is there anything that this test covers that isn't in the generated test?

Reply:
No, but the next PR will delete this test + add an _update test so I wanted to keep the file around (I *think* it would lose history downstream if I removed and then readded it which while not a big deal I prefer to keep)